transport/src/main/java/io/netty/channel/ChannelOutboundBuffer.java

```diff
@@ -829,4 +810,25 @@ Entry recycleAndGetNext() {
             return next;
         }
     }
+
+    private static final class ChannelWritabilityChangedTask implements Runnable {
+        private final Channel channel;
+        private boolean writable = true;
+
+        ChannelWritabilityChangedTask(Channel channel) {
+            this.channel = channel;
+        }
+
+        @Override
+        public void run() {
+            if (channel.isActive()) {
+                boolean newWritable = channel.isWritable();
+
+                if (writable != newWritable) {
```

Comment:
Yeah, that's why I removed 'boolean writability' parameter from `channelWritabilityChanged()` handler method. We are OK to fire more often than ideal.